packages/concerto-analysis/src/comparers/class-declarations.ts

```diff
@@ -54,11 +54,17 @@ const classDeclarationTypeChanged: ComparerFactory = (context) => ({
         if (aType === bType) {
             return;
         }
-        context.report({
```

Comment:
The existing type-change report shouldn't be removed. Instead, the `if` statement above should be inverted (`aType !== bType` or whatever) and the report should be put in there.